### PR TITLE
feat: openapi - BL21-107

### PR DESCRIPTION
**Pull request by mrnvr**
use openapitools/openapi-generator instead of swaggerapi/swagger-codegen
https://keyclic.atlassian.net/browse/BL21-107

### CODE REVIEW

File: bin/generate [hunk 1]
```diff
@@ -61,11 +57,10 @@ sed -i 's/List.listFromJson//g' lib/model/*
 
 # change some files to match requirements of pubspec.yaml specifications
 cp ../LICENSE .
-mv docs doc
-mv .analysis_options analysis_options.yaml
```
Comment (kevin-lot) [inline bin/generate:65]: Does generator generate right files or directory name ?
We need them to upload package on pub without issues.

File: dart-templates/class.mustache [hunk 2]
```diff
@@ -10,7 +10,7 @@
   {{/allowableValues}}
 {{/vars}}
 
-class {{classname}} {{#interfaces}}extends {{interfaces.0}} {{/interfaces}}{
+class {{classname}} {{#interfaces}}{{#-first}}implements {{/-first}} {{.}}{{^-last}}, {{/-last}} {{/interfaces}}{
```
Comment (kevin-lot) [inline dart-templates/class.mustache:13]: Are we implementing real interfaces ?

Comment (mrnvr) [reply dart-templates/class.mustache:13]: no

File: bin/generate (hunk 1 above)
Comment (mrnvr) [reply bin/generate:65]: yes

**Pull request closed without merging**